### ISSUE TITLE
Add term thickened cortex of the tibia

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10450]
**Preferred term label:**
Thickened cortex of the tibia

**Synonyms**
Broad cortex of tibia - Cortical thickening of the tibia - Thickened cortices of tibia

**Definition (free text, please give PubMed ID)**
Abnormal thickening of the cortex of the tibia
PMID24389367

**Parent term (use hpo.jax.org/app)**
Thickened cortex of long bones HP:0000935
Abnormal tibia morphology HP:0002992

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Caffey Disease (omim114000), for example: PMID21249479, PMID24390061

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000809
- label: Thickened tibial cortex
- parents: [Thickened cortex of long bones (HP:0000935)](https://hpo.jax.org/app/browse/term/HP:0000935) | [Abnormal tibia morphology (HP:0002992)](https://hpo.jax.org/app/browse/term/HP:0002992)
- pmid: PMID:21249479|PMID:24390061
- def: Abnormally increased thicknes of the cortical layer of the tibia.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10450
- synonyms: Thickened cortex of the tibia(NONE)
